app/assets/stylesheets/help.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Help controller here.
```

Comment:
Check in with the other teams and see if anyone has styling suggestions. You CAN just use Bootstrap (https://getbootstrap.com/), and that's a good place to start.